php/integration/class-gcp-storage.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * This file is responsible for getting files from Google Cloud Storage.
+ *
+ * @package WP_Tide_API
+ */
+
+namespace WP_Tide_API\Integration;
+
+use Google\Cloud\Storage\StorageClient;
+use WP_Tide_API\Base;
+
+/**
+ * Class GCP_Storage
+ */
+class GCP_Storage extends Base {
```

Comment:
Seems we could name this just `GCP` since it's namespaced and the `S3` and `Local` or prefix with `Storage`, but that seems redundant.